packages/react/src/components/InAppMessaging/CloseIconButton/types.ts

```diff
@@ -0,0 +1,15 @@
+import { LinkProps } from '../../../primitives';
+
+export interface CloseIconButtonProps extends Omit<LinkProps, 'children'> {
+  /**
+   * @description
+   * Configures the accessible label for the close iconButton
+   */
+  dismissButtonLabel?: string;
+
+  /**
+   * @description
+   * Function called when close iconButton is clicked
+   */
+  onClick?: React.MouseEventHandler<HTMLButtonElement>;
```

Comment:
I think if we are extending ButtonProps we no longer need to declare this.

Reply:
Yep, missed updating from Link to Button props